app/models/article.rb

```diff
@@ -156,7 +156,11 @@ class Article < ApplicationRecord
   serialize :cached_user
   serialize :cached_organization
 
-  pg_search_scope :search_reading_list,
+  # TODO: [@rhymes] Rename the article column and the trigger name.
+  # What was initially meant just for the reading list (filtered using the `reactions` table),
+  # is also used for the article search page.
+  # The name of the `tsvector` column and its related trigger should be adapted.
+  pg_search_scope :search_articles,
                   against: :reading_list_document,
```

Comment:
Basically `reading_list_document` is the exact combination of fields we need to search against for the search page, so I'm just renaming the method. Since renaming the trigger and column will take a bit of coordination I'm going to avoid it for now. 

I should have anticipated this scenario but I left `search articles` for last and I didn't know in time (I assumed I needed a different set of fields)